docs/deployment/add-node-to-cluster.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Manually adding a node to an existing cluster
+description: How to add a node to an existing cluster without the support of Gardener
```

Comment:
I am not sure this is even working as MCM has a logic to clean-up such manually joined nodes, AFAIK.
@gardener/mcm-maintainers any feedback on this guide will be valuable.

I am also not sure it should be in g/g. For sure, it must not be in `docs/deployment` which is about deploying components of Gardener.

Reply:
Hi @vpnachev,

You are right. MCM does have a logic to orphan collect such VMs/nodes. However, these VMs/nodes are identified by certain predefined tags. So yeah, non Gardener managed nodes can exist in the cluster.

Reply:
Then what do you suggest about this document?

Reply:
Move it to https://github.com/gardener/machine-controller-manager/